classes/task/UpdateRorRegistryDataset.php

```diff
@@ -116,330 +112,131 @@ public function getName(): string
     /** @copydoc ScheduledTask::executeActions() */
     public function executeActions(): bool
     {
-        $this->fileManager = new PrivateFileManager();
-        $pathZipDir = $this->fileManager->getBasePath() . DIRECTORY_SEPARATOR . $this->prefix;
-        $pathZipFile = $pathZipDir . '.zip';
+        $downloadUrl = $this->getDownloadUrl();
+        if (empty($downloadUrl)) {
+            UpdateRorRegistryDataset::log(
```

Comment:
here we use log function and below also addExecutionLogEntry. Could we here only use addExecutionLogEntry?